eng/common/scripts/Package-Properties.ps1

```diff
@@ -415,10 +415,12 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
                     $directory = [System.IO.Path]::GetDirectoryName($ciYml).Replace("`\", "/")
```

Comment:
orthogonal just noticed while reading. Given the first think you do is resolve the full path to the triggering files each time you use them it might be worth doing that once when you capture them.

Reply:
Ok. `changedFiles` will always be in `unix` style. `TriggeringPaths` will _also_ always be normalized when we populate it for `artifactdetails`. We don't need these `replace` at all aside from when we create the initial `filePath` from `Join-Path` at the top of this function.